mitodl/open-discussions
# Fix comment dialog dialog bug and refactor PostPage_test

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #1765 
Fixes #706 

#### What's this PR do?
Fixes the dialog not closing bug and also refactors tests on PostPage. There should be new tests for the bug and other related functionality.

#### How should this be manually tested?
Delete a comment. Click the accept button on the confirmation dialog. The comment should be deleted and the dialog should disappear.
